check-peak-detection.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+for i in $(grep cxidb $1 | grep ^Image | cut -f 3 -d " ")
+do
+  echo $i
+  python hdfsee.py  ~/$i  -g $2 -p $1
+done
```

Comment:
can you add a description of what this does and what inputs are expected?

Reply:
done in: cb46096f263b473b6239525f2f70b23dd1e591e3